### PR TITLE
Use entity ID badges for team ID's everywhere so we can show the external as well as the internal ID and make it look better.

### PR DESCRIPTION
**Pull request by nickygerritsen**
Also use the badges on the submission detail page for user, contest, problem and language.

With thanks to @meisterT .

Screenshots on request in Slack since data is officially still secret.

### CODE REVIEW

File: webapp/templates/jury/submission.html.twig [hunk 1]
```diff
@@ -123,13 +124,15 @@
...
                 {% endif %}
+                {{ submission.problem | entityIdBadge('p') }}
             </a>
         </span>
 
         <span>
             <i class="fas fa-comments" title="Language:"></i>
             <a href="{{ path('jury_language', {'langId': submission.language.langid}) }}">
                 {{ submission.language.name }}
+                {{ submission.language | entityIdBadge }}
```
Comment (tuupke) [inline webapp/templates/jury/submission.html.twig:135]: Maybe for consistency sake we should prefix the language id too

**Review by tuupke — APPROVED** [review]
Looks good, one very very minor nit. The language does not having a prefix (I remember seeing it in another place, but now cannot seem to find me) bugs me. If we can agree on what to prefix it with it might be nice to include it now.

File: webapp/templates/jury/submission.html.twig (hunk 1 above)
Comment (nickygerritsen) [reply webapp/templates/jury/submission.html.twig:135]: We never did this since language id’s are strings. lpy3 looks weird imho

Comment (tuupke) [reply webapp/templates/jury/submission.html.twig:135]: I know, that is why it is a very very minor nit.

**Review by meisterT — APPROVED** [review]
I think the unit test fails until you rebase this on current head of main

**Comment by nickygerritsen** [comment]
> I think the unit test fails until you rebase this on current head of main

It also found one actual stupid thing I did. Should now be fixed so if CI passes I will squash and merge.

**Review by meisterT — APPROVED** [review]

**Pull request merged**